packages/@tinacms/api-git/src/file-writer.ts

```diff
@@ -25,13 +25,26 @@ let count = 0
 
 const MAX_BUILD_TIME = 1000
 
-export function writeFile(filepath: string, content: string) {
-  count++
+export function checkFilePathIsInGitRepo(filepath: string, repoAbsolutePath: string) {
+  const fullpath = path.resolve(filepath)
+  const repopath = path.resolve(repoAbsolutePath)
+  if (fullpath.includes(repopath)) {
```

Comment:
`fullPath.startsWith(repopath)` might be safer. Might have to strip any leading `/` though

Reply:
Sure, not issue with startsWith. Not sure issue with leading `/`? Can you be more specific what you think that can be issue, we can cover as test case.